types/money/index.d.ts

```diff
@@ -0,0 +1,87 @@
+// Type definitions for money 0.2
+// Project: https://github.com/openexchangerates/money.js
+// Definitions by: Ivan Ergunov <https://github.com/hozblok>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare namespace fx {
+    type Currency = string;
+
+    type Rates = Record<string, number | string>;
+
+    type Options = {
+        from?: Currency | null;
+        to?: Currency | null;
+    } | null;
```

Comment:
```suggestion
    } ;
```
IMO just set options param as `options?` (optional), there is no trace of explicit null, just expression check for undefined/nullish options:
https://github.com/openexchangerates/money.js/blob/master/money.js#L59